src/test/java/app/component/Core.java

```diff
@@ -1,7 +1,20 @@
 package app.component;
 
+import app.common.EnvUtil;
+import app.common.HttpClient;
+
 public class Core extends App {
+    private static final String CORE_API_TOKEN = EnvUtil.getEnv("core_api_token");
```

Comment:
For the environment variables, can they all be prefixed with `UID2_E2E_*`?

i.e. `UID2_E2E_CORE_API_TOKEN`

This should match the other env vars in the tests. The three env vars here should then be added in the Dockerfile as well

Reply:
And later on, we would need to add the env vars as arguments in the uid2-shared-actions repo as well in the run_e2e_tests action